tools/qtc_templates/kuesa/project/empty/wizard.json

```diff
@@ -29,7 +29,7 @@
             "trDisplayName": "Project Location",
             "trShortTitle": "Location",
             "typeId": "Project",
-            "data": { "trDescription": "This wizard creates a simple Qt-based console application." }
+            "data": { "trDescription": "This wizard creates a simple Qt QML Kuesa -based console application." }
```

Comment:
Since you're at it, it's not a console application. :-)

Reply:
done